docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -3,305 +3,983 @@ title: JavaScript Quickstart Guide
+    bottom: 0;
+    width: 100%;
+    padding: 15px;
+    justify-content: center;
+    z-index: 10;
+  }
+
+  .control-bar > *:not(:first-child) {
+    margin-left: 8px;
+  }
+
+  .btn-control {
+    font-size: 12px;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    border: 2px solid #37474f;
+    width: 64px;
+    height: 64px;
+    border-radius: 50%;
+    text-align: center;
+    background-color: #607d8b;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.4);
+    color: white;
+  }
 ```
 
 </Tab>
 
-![Join form](/guides/join-room.png)
+### Test the app
 
-Congratulations! **you now have an audio-only conference ready to use**. If you have a friend join from another device, you can have a nice friendly chat with them.
+To test the application, run the application using:
 
-### Leaving the room
+-   using Yarn `yarn parcel 100ms-test/index.html --port 3000` or
+-   when using npm, run `npx parcel index.html --port 3000`.
 
-Before we go ahead with adding video, let us add a way to leave the room as well. We can call the `leave` method on `hmsActions` to leave the room.
+This will start your development server in http://localhost:3000/. Open this in your browser to test the video conferencing app you created above.
 
-We'll set up the `leave` method to be called whenever the user closes the window or refreshes the tab.
+#### Fetch token to join the room
 
-```jsx:script.js section=LeavingTheRoom sectionIndex=1
-function leaveRoom() {
-    hmsActions.leave();
-}
+To test audio/video functionality, you need to connect to a 100ms room; please check following steps for the same:
+
+1. Navigate to your [100ms dashboard](https://dashboard.100ms.live/dashboard) or [create an account](https://dashboard.100ms.live/register) if you don't have one.
+2. Use the `Video Conferencing Starter Kit` to create a room with a default template assigned to it to test this app quickly.
+3. Go to [Rooms page](https://dashboard.100ms.live/rooms) in your dashboard, click on the `Room Id` of the room you created above, and click on the `Join room` button on the top right.
+4. You will see two 100ms demo URLs for two roles created when you deployed the starter kit; you can click on the 'key' icon to copy the token and use the same to join the room in your browser.
+
+    ![Join room key](/guides/join_room_key.png)
+
+Now, after you click `join`, you should be able to see yourself!
+
+<video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
+    <source src="/guides/peer-1.mp4" type="video/mp4" />
+</video>
+<br />
 
-window.onunload = leaveRoom;
+You can open the app in another tab in the browser to join as the second peer to check audio/video transactions between two or more peers.
+
+<video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
+    <source src="/guides/peer-2.mp4" type="video/mp4" />
+</video>
+<br />
+
+**We have a fully functional video conferencing application now** 🎉🚀
+
+## A review of the code
```

Comment:
this section's language seems to be in terms of building the quickstart step by step than reviewing the code

Reply:
we can also change the section title to "Building Step by Step"

Reply:
done